components/bandwidth/sources/new-incoming-sms/new-incoming-sms.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  name: "New Incoming SMS",
+  description: "Emits an event each time a `message-received` event is received at the source url",
```

Comment:
Add a `key` (`bandwidth-new-incoming-sms`) and a `version` (0.0.1) here, as well. See https://pipedream.com/docs/components/guidelines/#required-metadata